components/Front/Link.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Link } from '../../lib/routes'
+import { parse } from 'url'
+
+import { PUBLIC_BASE_URL } from '../../lib/constants'
+import ArticleLink from '../Feed/ArticleLink'
```

Comment:
Maybe an indicator that both Feed/ArticleLink and Front/Link should live in a dedicated components/Link/ namespace.